soc/arm/cypress/common/cypress_psoc6_dt.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 Linaro Ltd.
+ * Copyright (c) 2020 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief Cypress PSoC-6 MCU family devicetree helper macros
+ */
+
+#ifndef _CYPRESS_PSOC6_DT_H_
+#define _CYPRESS_PSOC6_DT_H_
+
+#include <devicetree.h>
+
```

Comment:
Add a comment here about the main "API" being `CY_PSOC6_IRQ_CONFIG`.

Reply:
Addressed in the docs.